code/modules/busy_space/organizations.dm

```diff
@@ -0,0 +1,469 @@
+		var/list/destination_types = list("dockyard", "station", "vessel", "waystation", "telecommunications satellite", "spaceport", "distress beacon", "anomaly", "colony", "outpost")
+		while(i)
+			destination_names.Add("a [pick(destination_types)] in [pick(star_names)]")
+			i--
+
+//////////////////////////////////////////////////////////////////////////////////
+
+// TSCs
+/datum/lore/organization/tsc/nanotrasen
+	name = "NanoTrasen Incorporated"
+	short_name = "NanoTrasen"
+	acronym = "NT"
+	desc = "NanoTrasen is one of the foremost research and development companies in SolGov space. \
+	Originally focused on consumer products, their swift move into the field of Phoron has lead to \
+	them being the foremost experts on the substance and its uses. In the modern day, NanoTrasen prides \
+	itself on being an early adopter to as many new technologies as possible, often offering the newest \
+	products to their employees. In an effort to combat complaints about being 'guinea pigs', Nanotrasen \
+	also offers one of the most comprehensive medical plans in SolGov space, up to and including cloning \
+	and therapy.\
+	<br><br>\
+	NT's most well known products are its phoron based creations, especially those used in Cryotherapy. \
+	It also boasts an prosthetic line, which is provided to its employees as needed, and is used as an incentive \
+	for newly tested posibrains to remain with the company."
+	history = "" // To be written someday.
+	work = "research giant"
+	headquarters = "Earth"
+	motto = ""
+
+	ship_prefixes = list("NSV" = "exploration", "NTV" = "hauling", "NDV" = "patrol", "NRV" = "emergency response")
+	// Note that the current station being used will be pruned from this list upon being instantiated
+	destination_names = list(
+		"NSS Exodus in Nyx",
+		"NCS Northern Star in Vir",
+		"NCS Southern Cross in Vir",
+		"NDV Icarus in Nyx",
+		"an asteroid orbiting Kara",
+		"an asteroid orbiting Rota",
+		)
+
+/datum/lore/organization/tsc/nanotrasen/New()
+	..()
+	spawn(1) // BYOND shenanigans means using_map is not initialized yet.  Wait a tick.
+		// Get rid of the current map from the list, so ships flying in don't say they're coming to the current map.
+		var/string_to_test = "[using_map.station_name] in [using_map.starsys_name]"
+		if(string_to_test in destination_names)
+			destination_names.Remove(string_to_test)
+
+
+
+/datum/lore/organization/tsc/hephaestus
+	name = "Hephaestus Industries"
+	short_name = "Hephaestus"
+	acronym = "HI"
+	desc = "Hephaestus Industries is the largest supplier of arms, ammunition, and small millitary vehicles in Sol space. \
+	Hephaestus products have a reputation for reliability, and the corporation itself has a noted tendency to stay removed \
+	from corporate politics. They enforce their neutrality with the help of a fairly large asset-protection contingent which \
+	prevents any contracting polities from using their own materiel against them. SolGov itself is one of Hephastus� largest \
```

Comment:
Actually in this instance "materiel" is correct as it is referring to military equipment.